Source/WebCore/html/DateTimeFieldsState.h

```diff
@@ -43,6 +43,7 @@ struct DateTimeFieldsState {
     }
 
     std::optional<unsigned> year;
+    std::optional<unsigned> week;
```

Comment:
this is the most nit-picky thing ever, but I think this should be under month, since the order of these seems to be largest unit first